Tests/GRPCTests/GRPCReflectionServiceTests/ReflectionServiceIntegrationTests.swift

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright 2023, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+import GRPC
+import GRPCReflectionService
+import NIOPosix
+import SwiftProtobuf
+import XCTest
+
+@testable import GRPCReflectionService
+
+final class ReflectionServiceIntegrationTests: GRPCTestCase {
+  private var group: MultiThreadedEventLoopGroup?
+  private var server: Server?
+  private var channel: GRPCChannel?
+  private var protos: [Google_Protobuf_FileDescriptorProto] {
+    makeProtosWithDependencies()
+  }
```

Comment:
Because this is a computed property, each time we call it we call `makeProtosWithDependencies()` but the protos returned from it will never change. This can simply be `private let protos = makeProtosWithDependencies()`.